src/test/python/loader_native.py

```diff
@@ -0,0 +1,15 @@
+import sys
+sys.path.append("./generated")
```

Comment:
Do we need this? If so, can we resolve the full path (`os.path.abspath("generated")`) here? Otherwise the import directories move around if you change working directory throughout the library.

Reply:
this is just for tests, and frankly I need to both clean up the tests + move the generated code used by these tests out of the source tree